cfme/utils/appliance/__init__.py

```diff
@@ -482,8 +483,8 @@ def fix_httpd_issue(self, log_callback=None):
         """
                 full_cmd = """echo '{cmd}' >> {dst}""".format(cmd=exec_pre, dst=dst)
                 assert ssh.run_command(full_cmd).success
-                assert ssh.run_command('systemctl daemon-reload').success
-                assert ssh.run_command('systemctl restart httpd').success
+                self.daemon_reload.restart()
```

Comment:
The SystemdService method verifies the RC on restart and raises an exception on failure.

Reply:
why restart ? it will cause error.  I'd add reload method to SystemdService class instead

Reply:
OH sorry I totally mis-read this.